cli/src/semgrep/commands/ci.py

```diff
@@ -596,9 +596,11 @@ def ci(
             # console.print() would go to stderr; here we print() directly to stdout
             print(
                 json.dumps(
-                    scan_handler.ci_scan_results.to_json()
-                    if scan_handler.ci_scan_results
-                    else {},
+                    (
+                        scan_handler.ci_scan_results.to_json()
+                        if scan_handler.ci_scan_results
+                        else {}
+                    ),
```

Comment:
Any reason for the change here & above? Not sure if there's a autoformatter difference or something.